src/main/java/org/prebid/server/proto/response/BidderInfo.java

```diff
@@ -10,6 +10,8 @@
 
     boolean enabled;
 
+    boolean modifyingVastXmlAllowed;
```

Comment:
Minor. Pls declare the property after `gdpr` field.
Also, change `BidderInfo` constructor accordingly.